content/en/synthetics/browser_tests/actions.md

```diff
@@ -0,0 +1,114 @@
+description: Record Actions for a Synthetics Browser Test
+aliases:
+- /synthetics/browser_tests
+further_reading:
+- link: "synthetics/browser_tests/advance_options"
+  tag: "Documentation"
+  text: "Learn how to configure advance options for Actions"
+---
+
+## Overview
+
+Actions are a series of steps that you can record for a Browser Test, which you can then edit or build on. Certain actions can also be configured with [advanced options][1].
+
+### Assertion
+
+{{< img src="synthetics/browser_tests/browser_test_assertions.png" alt="Browser Test Assertion"  style="width:40%;">}}
+
+Assertions allow you to check whether an element, some content, or some text is available on the current page. You can also check whether a specific email was sent.
+
+The default timeout for each step is approximately 60 seconds. You can override the timeout with [advance options][2].
+
+| Assertion                                                 | Description                                                                                                                                                                             |
+|-----------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `Test that an element is present on the active page`      | Asserts that an element (such as a specific `span`, `div`, `h`, `a`, etc.) is present on the current page.                                                                              |
+| `Test an element's content`                               | Selects an element and checks if it contains a specific value. For instance, you could select a `div` and check whether it contains the word "hello".                                   |
+| `Test that some text is present on the active page`       | Asserts that some specific text is present on the current page.                                                                                                                         |
+| `Assert that some text is not present on the active page` | Asserts that some specific text is **NOT** present on the current page.                                                                                                                 |
+| `Test the content of the URL of the active page`          | Takes the URL of the last page that was interacted with, then asserts whether a specific value (`string`, `number`, `regex`) is present within it.                                      |
+| `Test that an email was received`                         | Asserts that an email was sent and whether specific values (`string`, `number`, `regex`) are present within the email subject or body. This assertion leverages  [email variables][10]. |
+
+[Advanced options][3] are also available for assertions.
+
+### Navigation
+
+The navigation action allows you to:
+
```

Comment:
Can we have a table for the three available navigation steps? Just like we have for assertions? To make the whole page more homogeneous

Reply:
Since these are step-by-step actions, I'm keeping as a bulleted list to be congruent with the rest of the docs.